src/appengine/handlers/cron/triage.py

```diff
@@ -14,12 +14,14 @@
 """Automated bug filing."""
 
 import datetime
+from functools import lru_cache
```

Comment:
Using lru_cache here can be problematic, because we don't know if a previous instance ran this, or is completely fresh. 

We should use our own cache which gets thrown away at the end of hte run.